notebook/templates/login.html

```diff
@@ -85,6 +85,22 @@
       <p>
         Cookies are required for authenticated access to notebooks.
       </p>
+      <h3>{% trans %}Setup a Password{% endtrans %}</h3>
```

Comment:
This whole block should be conditional on password not being set already. And there should probably be a `set_password_available` for explicit disabling, as well.